lib/contacts/publisher.rb

```diff
@@ -1,8 +1,8 @@
 module Contacts
   class Publisher
-    def self.publish(presenter)
+    def self.publish(base_path, presenter)
       presented = presenter.present
-      Contacts.publishing_api.put_content_item(presented[:base_path], presented)
+      Contacts.publishing_api.put_content_item(base_path, presented)
```

Comment:
this change seems to have made `Contacts::Publisher.publish` redundant. it only knows that it has to call `Contacts.publishing_api.put_content_item`.

should we get rid of `Contacts::Publisher` and use `Contacts.publishing_api.put_content_item(base_path, presented)` directly?

i don't see a case for extracting common publishing behaviour to anywhere other than the presenter or the model, so we should be good to clean this up. what say?